packages/nc-gui/components/smartsheet/column/LookupOptions.vue

```diff
@@ -83,15 +83,27 @@ const cellIcon = (column: ColumnType) =>
       <a-form-item class="flex w-1/2 pb-2" :label="$t('labels.links')" v-bind="validateInfos.fk_relation_column_id">
         <a-select
           v-model:value="vModel.fk_relation_column_id"
-          dropdown-class-name="!w-64 nc-dropdown-relation-table"
+          dropdown-class-name="!w-64 !rounded-md nc-dropdown-relation-table "
```

Comment:
unexpected space at the end

Reply:
**Fixed in below commit**

[fix : removed unwanted spaces](https://github.com/nocodb/nocodb/pull/7181/commits/4b8ee083f5689fd543f5a727c98a384e9f250525)
[4b8ee08](https://github.com/nocodb/nocodb/pull/7181/commits/4b8ee083f5689fd543f5a727c98a384e9f250525)